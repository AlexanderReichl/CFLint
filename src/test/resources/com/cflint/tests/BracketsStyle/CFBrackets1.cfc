component{
 
  function x()
  {
     var x = 1;
  }

}