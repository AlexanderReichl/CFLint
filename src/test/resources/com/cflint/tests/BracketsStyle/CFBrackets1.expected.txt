[ {
  "severity" : "INFO",
  "id" : "BRACKETS_STYLE",
  "message" : "BRACKETS_STYLE",
  "category" : "CFLINT",
  "abbrev" : "BS",
  "location" : {
    "file" : "",
    "fileName" : "",
    "column" : "0",
    "line" : "4",
    "message" : "Brackets should be formatted consistently.",
    "variable" : "",
    "expression" : ""
  }
} ]